packages/legacy/core/App/components/tour/TourOverlay.tsx

```diff
@@ -52,7 +52,7 @@ export const TourOverlay = (props: TourOverlayProps) => {
     const gapBetweenSpotAndTooltip = 50
     // if origin spot (ie. no spotlight)
     if (spot.x === 0 && spot.y === 0) {
-      const top = windowHeight / 5
+      const top = windowHeight / 6
```

Comment:
Magic number? Able to define as a const like you do for 50 above?

Reply:
I've made the variable name more descriptive and added a comment above it. I only use it in one place and it is just some relative spacing of one sixth the screen height so that the tooltip doesn't start at the very top of the screen. I'm not sure what I was name the six if I made it a const. Hopefully the descriptive var name and comment is enough